security-hub.tf

```diff
@@ -1,15 +1,13 @@
 module "monitoring_security_hub" {
-  source  = "dasmeta/monitoring/aws//modules/aws-security-hub-opsgenie"
-  version = "1.5.2"
+  source  = "dasmeta/monitoring/aws//modules/security-hub"
+  version = "1.19.5"
 
+  name = var.security_hub_name
```

Comment:
the name is module standard variable so lets have it placed with standard variables listing under `count`

Reply:
the value of name should be fetched from `var.alarm_actions.security_hub_alarms.name`, right?